executor/adapter.go

```diff
@@ -1700,6 +1751,21 @@ func GetResultRowsCount(stmtCtx *stmtctx.StatementContext, p plannercore.Plan) i
 	return rootStats.GetActRows()
 }
 
+// GetResultRowsCountForCopStats gets the count of the statement result rows of cop stats
+// Return -1 if no cop stats exists
+func GetResultRowsCountForCopStats(stmtCtx *stmtctx.StatementContext, planID int) int64 {
```

Comment:
It would be cleaner if you return pass/fail from the function.

Reply:
Get it~